tests/integration/ethernet_mtu_test.py

```diff
@@ -0,0 +1,109 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import copy
+
+import pytest
+
+from libnmstate import netapplier
+
+from .testlib import assertlib
+from .testlib import statelib
+from .testlib.statelib import INTERFACES
+
+
+def test_increase_iface_mtu():
+    desired_state = statelib.show_only(('eth1',))
+    eth1_desired_state = desired_state[INTERFACES][0]
+    eth1_desired_state['state'] = 'up'
+    eth1_desired_state['mtu'] = 1900
+
+    netapplier.apply(copy.deepcopy(desired_state))
+
+    assertlib.assert_state(desired_state)
+
+
+def test_decrease_iface_mtu():
+    desired_state = statelib.show_only(('eth1',))
+    eth1_desired_state = desired_state[INTERFACES][0]
+    eth1_desired_state['state'] = 'up'
+    eth1_desired_state['mtu'] = 1000
+
+    netapplier.apply(copy.deepcopy(desired_state))
+
+    assertlib.assert_state(desired_state)
+
+
+def test_upper_limit_jambo_iface_mtu():
+    desired_state = statelib.show_only(('eth1',))
+    eth1_desired_state = desired_state[INTERFACES][0]
+    eth1_desired_state['state'] = 'up'
+    eth1_desired_state['mtu'] = 9000
+
+    netapplier.apply(copy.deepcopy(desired_state))
+
+    assertlib.assert_state(desired_state)
+
+
+def test_increase_more_than_jambo_iface_mtu():
+    desired_state = statelib.show_only(('eth1',))
+    eth1_desired_state = desired_state[INTERFACES][0]
+    eth1_desired_state['state'] = 'up'
+    eth1_desired_state['mtu'] = 10000
+
+    netapplier.apply(copy.deepcopy(desired_state))
+
+    assertlib.assert_state(desired_state)
+
+
+def test_decrease_to_zero_iface_mtu():
+    desired_state = statelib.show_only(('eth1',))
+    origin_desired_state = copy.deepcopy(desired_state)
+    eth1_desired_state = desired_state[INTERFACES][0]
+    eth1_desired_state['state'] = 'up'
+    eth1_desired_state['mtu'] = 0
+
+    with pytest.raises(Exception):
```

Comment:
The expected exception should be more specific than `Exception` to ensure that this does not mask an unexpected exception.